### PR TITLE
feat: upgrade to the single-spa 5.6.0

### PR DESCRIPTION
**Pull request by StyleT**

### CODE REVIEW

File: ilc/client/isActiveFactory.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {triggerAppChange} from 'single-spa';
+import {triggerAppChange} from './navigationEvents';
```
Comment (oleh-momot) [inline ilc/client/isActiveFactory.js:1]: Do we have here events setting up twice?
Look at `ilc/client` line number 1

File: ilc/client.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import './client/navigationEvents';
```
Comment (oleh-momot) [inline ilc/client.js:1]: It is a little bit tricky
I think that would be easy just to import `setupEvents` here

File: ilc/client/isActiveFactory.js (hunk 1 above)
Comment (StyleT) [reply ilc/client/isActiveFactory.js:1]: Nope, due to the cache of the imports. Files are executed only once

File: ilc/client.js (hunk 2 above)
Comment (StyleT) [reply ilc/client.js:1]: The idea is to have single entry point to this "module"

**Review by oleh-momot — APPROVED** [review]

**Review by nightnei — APPROVED** [review]

**Pull request merged**